test/extended/testdata/bindata.go

```diff
@@ -13900,7 +13908,8 @@ var _examplesDbTemplatesRedisEphemeralTemplateJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Password: ${REDIS_PASSWORD}\n Connection URL: redis://${DATABASE_SERVICE_NAME}:6379/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/redis-container/blob/master/3.2.",
   "labels": {
-    "template": "redis-ephemeral-template"
+    "template": "redis-ephemeral-template",
+    "app": "redis-ephemeral-template"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.